client/src/components/NewBoardGameForm.js

```diff
@@ -62,6 +67,12 @@ const NewBoardGameForm = props => {
 		addNewBoardGame()
 	}
 
+	useEffect(() => {
+		if (role === "user" || role === "admin") {
+			setShowButton(true)
+		}
+	})
```

Comment:
Nick's comment still stands - we should remove the useEffect and just have the component handle this logic. Reach out if you have any questions about this